src/open_inwoner/configurations/models.py

```diff
@@ -209,6 +209,13 @@ class SiteConfiguration(SingletonModel):
         blank=True,
         help_text=_("Visiting intro text on the footer section."),
     )
+    footer_visiting_phonenumber = models.CharField(
```

Comment:
- You can add a phone validator here. We use it in this kind of fields (`utils/validators/validate_phone_number`) to prevent people from entering unwanted values.
- `max_length` should have the necessary value (you can put 15 here)